mdoc/Mono.Documentation/Updater/Formatters/CSharpFullMemberFormatter.cs

```diff
@@ -30,6 +30,7 @@ protected override StringBuilder AppendNamespace (StringBuilder buf, TypeReferen
         protected virtual string GetCSharpType (string t)
         {
             // make sure there are no modifiers in the type string (add them back before returning)
+            // TODO: Do we really need this code? When does it work? What is the modifier?
```

Comment:
yes, this is necessary. it's related to `modreq` and `modopt` modifiers ... mainly added by c++ compilers, but are not supported in any way in C# currently. You can remove this comment :)